README.md

```diff
@@ -63,6 +67,51 @@ Example:
 }
 ```
 
+### TypeChunksFromFile
+
+Type text at the current cursor position of the currently open file. Text is read from the file indicated by the path, relative to the `.presentation-buddy` folder. The input file will be split into chunks based on the supplied settings. 
+
+By default, the input will be split on newlines (`\n`). To disable this, set `waitAfterNewLines` to `false`.
+
+Windows-style `\r\n` line endings will be converted to `\n` before the file is processed.
+
+Presentation Buddy will:
+
+* `wait` after typing any string matching a supplied `waitAfter` argument
+* `wait` **instead of** typing any string matching a supplied `waitInsteadOf` argument
+* Skip any line containing any string matching a supplied `skipLinesContaining` argument.
+
+#### Example:
+
+This will type the contents of `chunks-exampe.js` into the current window, one line at a time.
+
+Any line containing the comment `//skip` will be skipped. Presentation Buddy will pause after typing an opening curly bracket `{`, a dot `'`, a lambda function symbol `  =>  `, or an assignment `  =  `. If the input contains the string `/*WAIT*/`, this will pause Presentation Buddy but the `/*WAIT*/` won't be copied to the output.
```

Comment:
```suggestion
Any line containing the comment `//skip` will be skipped. Presentation Buddy will pause after typing an opening curly bracket `{`, a dot `.`, a lambda function symbol `  =>  `, or an assignment `  =  `. If the input contains the string `/*WAIT*/`, this will pause Presentation Buddy but the `/*WAIT*/` won't be copied to the output.
```